dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/MetricsCollectController.java

```diff
@@ -34,16 +33,13 @@
 import org.apache.dubbo.metadata.definition.model.FullServiceDefinition;
 import org.apache.dubbo.metadata.report.identifier.MetadataIdentifier;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```

Comment:
please install codestyle plugin in your ide
class import with wildcard is not allowed in Dubbo

you can check this in https://github.com/apache/dubbo/blob/master/CONTRIBUTING.md

Reply:
Ops.. thanks for your comment. changes have been revoked.